app/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Auth\LoginRequest;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
+
+class LoginController extends Controller
+{
+    public function index()
+    {
+
+        return Inertia::render('Auth/Login');
+
+    }
+
+    public function login(LoginRequest $request)
+    {
+
+        // validate email and password through LoginRequest
+        $validated = $request->validated();
+
+        // // check if user exists
+        if (Auth::attempt($validated, $request->only('remember_me'))) {
+
+            // Generate session
+            $request->session()->regenerate();
+
+            //     // redirect to platform`s dashboard page
+            return redirect()->intended('/dashboard');
+        }
+
+        // // If user does not exist
+        return back()->withErrors([
+            'error' => 'The provided credentials do not match our records.',
+        ]);
+
+    }
+
+    public function logout()
+    {
+
+        Auth::logout();
+
```

Comment:
Besides logging out, you are missing regeneration of the token and invalidating the old session.